### DISCUSSION TITLE
Getting a model from an imported module

### DISCUSSION THREAD

**Discussion by epreciadoNxA** [swaggo/swag#1284] [General]
Hi I was wondering if it is possible to reference an external model for it to show in the swagger? 

I have a private repository imported into my go modules in there I have a model defined for a specific error message to be returned. In the comment I am able to have it highlighted in the IDE but when I run swag init it fails to recognize the object.

**Comment by bekabaz** [comment]
> Hi I was wondering if it is possible to reference an external model for it to show in the swagger?
> 
> I have a private repository imported into my go modules in there I have a model defined for a specific error message to be returned. In the comment I am able to have it highlighted in the IDE but when I run swag init it fails to recognize the object.

Try running `swag init --parseDependency --parseInternal --parseDepth 1` which will allow it to read external imports.
